onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -122,6 +122,13 @@ void SplitTestAxis0EqualSplit(bool use_opset_13 = false) {
              {kTensorrtExecutionProvider},  // is_tensorrt_supported
              false,                         // expect_failure
              use_opset_13);                 // split_as_input
+#ifdef USE_COREML
+  RunTest<T>(axis, {}, input, outputs,
+             // TensorRT parser: Assertion failed: axis != BATCH_DIM
+             {kTensorrtExecutionProvider},  // is_tensorrt_supported
+             false,                         // expect_failure
+             use_opset_13 = true);          // split_as_input
```

Comment:
this is assigning `use_opset_13` to true and then returning the result of that assignment, true.

is the intent just to pass true?

Reply:
updated.